syftlib/src/main/java/org/openmined/syft/networking/requests/HttpAPI.kt

```diff
@@ -23,22 +23,34 @@ import retrofit2.http.Part
 import retrofit2.http.Query
 import retrofit2.http.Streaming
 
+/**
+ * HttpAPI interface is used to implement http and socket connection API to PyGrid Server.
+ */
 interface HttpAPI : CommunicationAPI {
 
+    /**
+     * Checks connection speed by ping to PyGrid Server.
+     */
```

Comment:
Adding params will also help in cases where they are not very descriptive

Like random, is_ping, worker_id